docs/tap_hold.md

```diff
@@ -198,6 +198,38 @@ bool get_retro_tapping(uint16_t keycode, keyrecord_t *record) {
 }
 ```
 
+## Retro Shift
+
+Holding and releasing a Tap Hold key without pressing another key will result in only the hold. With Retro Shift enabled this action will also produce a shifted version of the tap keycode on release.
+
+This is a supplement to [Auto Shift](feature_auto_shift.md), which does not support Tap Hold.  Auto Shift will be enabled automatically if it is not already, but you should read the Auto Shift documentation and configure the Auto Shift timeout matching the tapping term.  All additional Auto Shift defines are respected, including keyrepeat.
```

Comment:
If this is using functions from `process_auto_shift` then that's a problem

Reply:
I'm not sure I follow. Do you mean that it should be a separate feature that can work without Auto Shift enabled?

Reply:
Wait, so this does require autoshift? 

If so, this documentation should be in that documentation, and not here.   Because otherwise, it sounds like this is separate functionality.  Hence my comments.